style.css

```diff
@@ -0,0 +1,78 @@
+
+.container {
+    color: white;
+    margin: 0 auto;
+}
+
+header {
+  background-color: #ebebeb;
+  padding: 30px;
+  text-align: center;
+  font-size: 35px;
+  color: white;
+  border-style: solid;
+}
+
+Nav {
+    background-color: #ebebeb;
+    padding: 30px;
+    text-align: center;
+    font-size: 35px;
+    color:white;
+    border-style: solid;
+}
+
+body {
```

Comment:
You can also add `margin: auto` styling to help center the whole body on your page.